Python/tdw/tdw_utils.py

```diff
@@ -379,7 +379,7 @@ def get_depth_values(image: np.array) -> np.array:
         """
 
         # Convert the image to a 2D image array.
-        image = np.array(Image.open(io.BytesIO(image)))
+        image = np.array(Image.open(io.BytesIO(image)), dtype=np.uint32)

```

Comment:
Does it make more sense to use np.float? Since depth is a real value.

Reply:
@cffan There's no difference.

```python
img_0 = np.array(Image.open(io.BytesIO(image)), dtype=np.float)
depth_0 = np.array((img_0[:, :, 0] * 256 * 256 + img_0[:, :, 1] * 256 + img_0[:, :, 2]) / (256 * 256 * 256) * 100.1)

img_1 = np.array(Image.open(io.BytesIO(image)), dtype=np.uint32)
depth_1 = np.array((img_1[:, :, 0] * 256 * 256 + img_1[:, :, 1] * 256 + img_1[:, :, 2]) / (256 * 256 * 256) * 100.1)

print(np.linalg.norm(depth_0 - depth_1))
```

Result: 0.0